adam-python/src/bdgenomics/adam/test/adamContext_test.py

```diff
@@ -0,0 +1,116 @@
+#
+# Licensed to Big Data Genomics (BDG) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The BDG licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+from bdgenomics.adam.adamContext import ADAMContext
+from bdgenomics.adam.test import SparkTestCase
+
+
+class ADAMContextTest(SparkTestCase):
+
+
+    def test_load_alignments(self):
+        
+        testFile = self.resourceFile("small.sam")
+        ac = ADAMContext(self.sc)
+        
+        reads = ac.loadAlignments(testFile)
+
+        self.assertEqual(reads.toDF().count(), 20)
+        self.assertEqual(reads._jvmRdd.jrdd().count(), 20)
```

Comment:
Good Q, will check. I don't believe so, but not 100%. I think I should remove the `def rdd():` actually. There's a serialization issue crossing Java to Python with Avro objects in PySpark.